app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallScreen.kt

```diff
@@ -105,7 +117,13 @@ private fun CallingControls(
             .padding(0.dp, dimensions().spacing16x, 0.dp, 0.dp)
     ) {
         MicrophoneButton(ongoingCallState.isMuted) { onMuteOrUnMuteCall() }
```

Comment:
same here do not need to pass a lambda you could do it like this ; 
MicrophoneButton(onGiongCallState.isMuted,onMuteOrUnMuteCall)

Reply:
up